code/lib/builder-vite/src/plugins/mdx-plugin.ts

```diff
@@ -46,33 +30,16 @@ export function mdxPlugin(options: Options): Plugin {
         },
       });
 
-      const mdxCode = String(
+      const code = String(
         await compile(src, {
           skipCsf: !isStorybookMdx(id),
           ...mdxLoaderOptions,
```

Comment:
You're missing `jsxOptions`

Reply:
Yeah, I recon we want to get them from a similar place as mdxCompilerOptions: from a preset value